linera-indexer/lib/src/rocks_db.rs

```diff
@@ -13,6 +13,7 @@ use linera_views::{
 use std::path::PathBuf;
 
 #[derive(clap::Parser, Clone, Debug)]
+#[command(version = clap::crate_version!())]
```

Comment:
This one does nothing, does it?

Reply:
No this is actually important — unlike `structopt`, without this `clap` won't give us a `--version` flag (and the tests will fail, since we check it)!